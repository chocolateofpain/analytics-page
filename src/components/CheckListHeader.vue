<template>
  <thead class="table-header">
    <th>type</th>
    <th>name</th>
    <th class="align-center">success</th>
    <th class="align-center">avg</th>
    <th class="align-center">p95</th>
    <th class="align-center">p99</th>
  </thead>
</template>

<script>
export default {
  name: 'CheckListHeader'
}
</script>

<style scoped>
.table-header {
  border-bottom: 1px solid #000000;
  height: 28px;
  text-transform: uppercase;
}
</style>
