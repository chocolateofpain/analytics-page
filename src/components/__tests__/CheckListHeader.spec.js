import { mount } from '@vue/test-utils'
import CheckListHeader from '../CheckListHeader.vue'

describe('CheckListHeader', () => {
  it('should be defined', () => {
    expect(CheckListHeader).toBeDefined()
  })

  describe('snapshots', () => {
    it('should render component', () => {
      const component = mount(CheckListHeader)
      expect(component).toMatchSnapshot()
    })
  })
})
