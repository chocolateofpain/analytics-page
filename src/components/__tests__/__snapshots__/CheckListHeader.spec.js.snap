// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckListHeader snapshots should render component 1`] = `
<thead class="row-height table-header">
  <th class="type-col">type</th>
  <th class="name-col">name</th>
  <th class="data-col">success</th>
  <th class="data-col">avg</th>
  <th class="data-col">p95</th>
  <th class="data-col">p99</th>
</thead>
`;
